examples/html-css/README.md

```diff
@@ -0,0 +1,9 @@
+## Example: Pure HTMS/CSS
+
+This shows the usage of CMS design system components with a pure HTML/CSS set up. Simply include the CSS file in the HTML document.
+
+### Usage
+
+```html
+<link rel="stylesheet" href="packages/design-system/dist/css/index.css" />
```

Comment:
For the form HTML/CSS example, the images are not showing for the error message or the checkboxes or radios

Reply:
This is also happening with the alert on the article example

Reply:
Have you `yarn install` & `yarn build` the branch?

Reply:
That did the trick - We might want to mention the need to run `yarn install` and `yarn build` to get things to work

Reply:
I dont think this needs to be mentioned because the path in the HTML/CSS use case will be different for every app. We have it setup here directing to our packages, but thats not relevant to most users, nor would users be probably testing it via our github repo

Reply:
if anything, i would prefer this path to be replaced with something more realistic like:
```
href="vendor/cmsgov-design-system-2.0.0/index.css
```
and then maybe add a note that the path should go to whereever the CMSDS css file is stored in their project